packages/ckeditor5-ckbox/src/ckboxconfig.ts

```diff
@@ -137,8 +137,131 @@ export interface CKBoxConfig {
 	 * Defaults to {@link module:utils/locale~Locale#uiLanguage `Locale#uiLanguage`}
 	 */
 	language?: string;
+
+	/**
+	 * This option allows opening CKBox in dialog mode. It takes a boolean value or
+	 * a configuration object with the following attributes:
+	 */
+	dialog?: CKBoxDialog;
+
+	/**
+	 * Allows setting custom icons for categories.
+	 */
+	categories?: CKBoxCategories;
+
+	/**
+	 * Configures the view of CKBox.
+	 */
+	view?: CKBoxView;
+
+	/**
+	 * Configures when dialog should be minimized and hidden.
+	 */
+	upload?: CKBoxUpload;
+
+	/**
+	 * Specifies the file extensions considered valid for user interaction. Whith this
+	 * option developers can restrict user interaction to only those assets whose file
+	 * extensions match those listed in the array. Assets whose file
+	 * extensions are not listed in the choosableFileExtensions array are
+	 * automatically disabled within the CKBox interface.
+	 *
+	 * ```ts
+	 * const ckboxConfig = {
+	 *		choosableFileExtensions: ['jpg', 'png']
+	 * };
+	 * ```
+	 */
+	choosableFileExtensions?: Array<string>;
 }
 
+export type CKBoxDialog = {
```

Comment:
Why `export type` if the config uses `export interface`? Seems like an unnecessary inconsistency.

Reply:
Sure, I just followed the way we do it in other places, like `export-pdf`, so here we declare an **interface**:
https://github.com/cksource/ckeditor5-commercial/blob/master/packages/ckeditor5-export-pdf/src/exportpdf.ts#L92
but option of the config is **type**:
https://github.com/cksource/ckeditor5-commercial/blob/master/packages/ckeditor5-export-pdf/src/exportpdf.ts#L219

But as I see in other places is done with `interfaces` indeed.